front/src/Phaser/UserInput/UserInputManager.ts

```diff
@@ -65,14 +66,21 @@ export class UserInputManager {
 
     clearAllKeys(){
```

Comment:
We should rename `clearAllKeys` to `disableControls` and `restoreAllKeys` to `restoreControls`.
That seems a better more meaningful name.